thanos/charts/templates/query/sd-config.yaml

```
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "release.name" . }}-sd-config
```

The ConfigMap is missing the default labels. 
```suggestion
metadata:
  labels:
    {{- include "plugin.labels" . | nindent 4 }}
    {{- include "thanos.labels" . | nindent 4 }}
    {{- if .Values.thanos.query.deploymentLabels }}
    {{ toYaml .Values.thanos.query.deploymentLabels | nindent 4 }}
    {{- end }}
  name: {{ include "release.name" . }}-sd-config
```